dev/api_core/code_samples/Python samples/Setting and Removing User Rights.md

```diff
@@ -0,0 +1,83 @@
+# Working with Setting and Removing User Rights
+
+In this code sample we will show how to set and remove User Rights. 
+
+## Required Modules
+
+- simplic
+  - [DependencyInjection](xref:PythonAPI.DependenyInjection)
+- [Simplic.Authorization](xref:Simplic.Authorization)
+  - [RowAccess](xref:Simplic.Authorization.RowAccess)
+  - [IAuthorizationService](xref:Simplic.Authorization.IAuthorizationService)
+
+## Sample
+
+
+# Python
+
+### Setting user rights
+```python
+# Import
+from simplic import DependencyInjection
+from Simplic.Authorization import RowAccess, IAuthorizationService
+
+	def set_document_access(self, document_id):
+        # Getting the Authorization Service
+		service = DependencyInjection.resolve(IAuthorizationService)
+		# Sample groupId's
+        gl = 7
+		accounting = 10
+        # Getting the Access of the given document_id
+        # First parameter (string): Table to query (IT_Document)
+		# Second parameter (string): Name of the id column (Guid)
+		# Third parameter (object): Actual rowId (document_id)
+		access_obj = service.GetAccessRights('IT_Document', 'Guid', document_id)
+
```

Comment:
Something with the formatting seems to be wrong. Please change gl --> user_group_1